packages/theme/src/typography/android.ts

```diff
@@ -83,6 +83,12 @@ export const androidTextTypeStyles: TextTypeStyles = {
     ...primaryBase,
     fontWeight: '500',
   },
+  'heading--medium': {
+    fontSize: 20,
+    lineHeight: 24,
+    letterSpacing: 0.5,
+    fontWeight: 'bold',
```

Comment:
Just a question from my side: is it odd having heading medium which is in fact not medium but bold? (I get that medium here means between normal heading and big, but might not be too intuitive)

Reply:
@ckbilstad

Reply:
True true, but i think medium is fine as of now seen in relation with the other variables and the context. However, naming might be something to look closer into if we get the chance to improve the design system.